queries/issue_assignee_query.py

```diff
@@ -0,0 +1,110 @@
+import logging
+import pandas as pd
+from db_manager.augur_manager import AugurManager
+from app import celery_app
+from cache_manager.cache_manager import CacheManager as cm
+import io
+import datetime as dt
+from sqlalchemy.exc import SQLAlchemyError
+
+QUERY_NAME = "ISSUE_ASSIGNEE"
+
+
+@celery_app.task(
+    bind=True,
+    autoretry_for=(Exception,),
+    exponential_backoff=2,
+    retry_kwargs={"max_retries": 5},
+    retry_jitter=True,
+)
+def issue_assignee_query(self, repos):
+    """
+    (Worker Query)
+    Executes SQL query against Augur database for contributor data.
+    Args:
+    -----
+        repo_ids ([str]): repos that SQL query is executed on.
+    Returns:
+    --------
+        dict: Results from SQL query, interpreted from pd.to_dict('records')
+    """
+    logging.warning(f"{QUERY_NAME}_DATA_QUERY - START")
+
+    if len(repos) == 0:
+        return None
+
+    query_string = f"""
+                    SELECT
+                        i.issue_id,
+                        r.repo_id as id,
+                        i.created_at as created,
+                        i.closed_at as closed,
+                        ie.created_at AS assign_date,
+                        ie.action AS assign,
```

Comment:
I'm guessing the action is 'assigned' or 'unassigned', so I think this variable name is somewhat confusing. Maybe "assignment_action" or something would be easier to understand in context in the processing logic.